lib/app/services/network.dart

```diff
@@ -89,7 +89,17 @@ class ServerError extends FancyException {
   final ErrorBody body;
 }
 
+// 3xx
+
+class MovedPermanentlyError extends ServerError {
+  MovedPermanentlyError(ErrorBody body, this.location)
+      : super(body, (context) => context.s.app_error_movedPermanently);
```

Comment:
I'm not sure whether this should actually have a user-facing string. The current localized values aren't really helpful for the user (in case they even understand them), and by default, the HTTP package will follow redirects (hence the optional parameter `followRedirects` to disable that behavior). So I don't think they will ever be shown and hence recommend to just throw an exception when accessing this value.